OpenDDSharp/OpenDDSharp.IdlGenerator/NativeIdlProject.vcxproj

```diff
@@ -204,10 +204,10 @@
   </ItemGroup>
   <Target Name="OpenDDSIdl" BeforeTargets="PreBuildEvent">
     <Message Text="Generating %(IdlFiles.Identity) code..." Importance="High" />
-    <Exec Command="cmd /c &quot;$(ACE_ROOT)\bin\tao_idl -Wb,pre_include=ace/pre.h -Wb,post_include=ace/post.h -I$(TAO_ROOT) -Sa -St -I$(DDS_ROOT) -Wb,export_macro=%(IdlFiles.Filename)IDL_Export -Wb,export_include=%(IdlFiles.Filename)IDL_Export.h &quot;%(IdlFiles.Identity)&quot;&quot;" IgnoreExitCode="false" ContinueOnError="false" />
-    <Exec Command="cmd /c &quot;$(DDS_ROOT)\bin\opendds_idl -Sa -St -Wb,export_macro=%(IdlFiles.Filename)IDL_Export &quot;%(IdlFiles.Identity)&quot;&quot;" IgnoreExitCode="false" ContinueOnError="false" />    
+    <Exec Command="cmd /c &quot;$(ACE_ROOT)\bin\tao_idl --idl-version 4 -Wb,pre_include=ace/pre.h -Wb,post_include=ace/post.h -I$(TAO_ROOT) -Sa -St -I$(DDS_ROOT) -Wb,export_macro=%(IdlFiles.Filename)IDL_Export -Wb,export_include=%(IdlFiles.Filename)IDL_Export.h &quot;%(IdlFiles.Identity)&quot;&quot;" IgnoreExitCode="false" ContinueOnError="false" />
+    <Exec Command="cmd /c &quot;$(DDS_ROOT)\bin\opendds_idl --idl-version 4 -Sa -St -Wb,export_macro=%(IdlFiles.Filename)IDL_Export &quot;%(IdlFiles.Identity)&quot;&quot;" IgnoreExitCode="false" ContinueOnError="false" />    
```

Comment:
`tao_idl` requires this flag, but `opendds_idl` defaults to idl 4, so it's not needed for it.